WordPress/gutenberg
# Docs: fix redirected links to user documents

Follow-up on: #49533
Related core ticket: [#57726 (Use the new /documentation/ URLs for HelpHub links) – WordPress Trac](https://core.trac.wordpress.org/ticket/57726)

## What?
This PR fixes a problem with the redirection of links to documentation for users.

## Why?

Just like the changes made in the core, I think all `/support/article/*` URLs should be redirected to `/documentation/article/` to avoid an extra redirect.

## How?

- Search Gutenberg projects with `https://wordpress.org/support/`.
- Open the found link in a browser.
- Updated the article that is being redirected.

## Testing Instructions

- Check changed files.
- Access the URL before the change.
- Confirm that you are redirected to the changed URL.
